lib/src/native/realm_core.dart

```diff
@@ -1371,7 +1377,9 @@ extension on Pointer<realm_value_t> {
       case realm_value_type.RLM_TYPE_BINARY:
         throw Exception("Not implemented");
       case realm_value_type.RLM_TYPE_TIMESTAMP:
-        throw Exception("Not implemented");
+        final seconds = ref.values.timestamp.seconds;
+        final nanoseconds = ref.values.timestamp.nanoseconds;
+        return DateTime.fromMicrosecondsSinceEpoch(seconds * 1000000 + nanoseconds ~/ 1000).toUtc();
```

Comment:
Looks kosher, but I notice a lot of failing tests.

Reply:
Looks like a bug in date serialization in Core (or a bug in our code). Investigating.